src/containers/SideNav/SideNav.js

```diff
@@ -52,6 +63,10 @@ function SideNav(props) {
 
   const { namespace } = match?.params || {};
 
+  const { data: extensions = [], refetch } = useExtensions();
+
+  useWebSocketReconnected(refetch, webSocketConnected);
```

Comment:
no need for this here since it's already handled in App

Reply:
Ok, so `extensions` should be passed in as a prop to SideNav from App?

Reply:
sorry just meant the websocket piece, they're already being refetched on reconnect in App, so we can just call the `useExtensions` hook here and will benefit from the caching etc. already being done by react-query